### ISSUE TITLE
Chrome 67 seems to increase First Visual Change and Speed Index

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/sitespeed.io#2069]
Yesterday we pushed latest Chrome on the test servers on dashboard.sitespeed.io, preparing for the next release. However the [alerts](https://dashboard.sitespeed.io/d/000000010/alerts?panelId=6&fullscreen&orgId=1) we have fired for the tests we run on Wikipedia. At the moment we also pushed Chromedriver (or at least something like one hour later). 

I reverted back to Chrome 66 (by switching the tags from chrome-67-firefox-61b5 to chrome-66-firefox-61b5  https://hub.docker.com/r/sitespeedio/webbrowsers) and the metrics went back to normal (keeping the new Chromedriver). So I'm pretty sure the it's just the browser that makes the difference (the only update on the container was [changing the version number](https://github.com/sitespeedio/docker-browsers/commit/846e841589847fc1741339a9cc3f7f41538e0bfc) so we pickup 67).

Let check the dashboards:
We run [the tests](https://dashboard.sitespeed.io/d/000000010/alerts?panelId=17&fullscreen&orgId=1) on a C4.large on AWS
![speedindex](https://user-images.githubusercontent.com/540757/40877177-9fe448b6-667c-11e8-9f5e-95b60097ca49.png)

On the same server we run tests on NYTimes, RyanAir and sitespeed.io. For those sites I cannot see the same thing but both NYTimes and RyanAir are really unstable

So can this be a problem only for Wikipedia +- the server where we run the tests? We also run tests on Digital Ocean (optimized droplets): I think I see the same thing there, but since the server is faster, the metrics is not changed as much.

We also run tests with WebPageReplay on the same server and I can see change there too but only for Wikipedia (not CNET or izettle that we also test):
![barack](https://user-images.githubusercontent.com/540757/40877284-8dff9004-667e-11e8-9398-79340d1919ab.png)

So to summarize: We see it on Wikipedia on different servers but not on the other sites we test.
We collect trace logs from each run so we could look into it to see what happens more in detail.

Here's one run before the change (Chrome 66):
https://results.sitespeed.io/en.wikipedia.org/2018-06-01-05-19-43/pages/en.wikipedia.org/wiki/Barack_Obama/

And Chrome 67:
https://results.sitespeed.io/en.wikipedia.org/2018-06-01-11-16-04/pages/en.wikipedia.org/wiki/Barack_Obama/

The change isn't super big but we picked it up on our alerts. And it seems we at least only can see it on Wikipedia URLs so it should be ok to push the new version (I'll update at work after the weekend and see if I see the same thing).

I'll push a update to the Docker container now to see if the metrics goes up again just to verify one more time.

**Comment by soulgalore** [comment]
I added a new annotation at [the dashboard](https://dashboard.sitespeed.io/d/000000044/page-timing-metrics?orgId=1) ... it will take a couple of minutes for the new container to build and then one hour or so until it's there on the test servers.

**Comment by soulgalore** [comment]
Yep it happened again so it looks like it is related to Chrome 67. I also looked at RUM first paint this time and I see the same pattern (=it's not only Visual Metrics):

![facebook-chrome67](https://user-images.githubusercontent.com/540757/40879297-fc3f9a7c-669d-11e8-94b4-b20836176d31.png)

The only thing changed between these are the Chrome version. I can only see the change for Wikipedia though. I've checked at work with the test we run with WebPageTest and the increase match pretty good going to 67: https://grafana.wikimedia.org/dashboard/db/webpagetest?panelId=10&fullscreen&orgId=1 (check last of May/first of June).

Ping @paulirish
